packages/tests/server/websockets.test.ts

```
@@ -1959,6 +1959,7 @@ describe('subscriptions with createCaller', () => {
       SubscriptionProcedure<{
         input: string | null | undefined;
         output: AsyncIterable<Message, void, any>;
+        errors: void;
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)